tests/integration/test_nitrate.py

```diff
@@ -0,0 +1,85 @@
+import yaml
+import shutil
+import tempfile
+from fmf import Tree
+from pathlib import Path
+from click.testing import CliRunner
+from requre import RequreTestCase
+import tmt.base
+import tmt.cli
+
+# Prepare path to examples
+TEST_DIR = Path(__file__).parent
+NITRATE_EXAMPLE = TEST_DIR.parent.parent / "examples" / "nitrate_cases"
+
+
+# general test plan for this component is: TP#29309
+class NitrateExport(RequreTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.tmpdir = Path(tempfile.mktemp())
+        shutil.copytree(NITRATE_EXAMPLE, self.tmpdir)
+
+    def tearDown(self):
+        shutil.rmtree(self.tmpdir)
+        super().tearDown()
+
+    def test_create(self):
+        fmf_node = Tree(self.tmpdir).find("/new_testcase")
+        self.assertNotIn("extra-nitrate", fmf_node.data)
+        test = tmt.base.Test(fmf_node)
+        test.export(format_='nitrate', create=True, general=True)
+        # reload the node data to see if it appear there
+        fmf_node = Tree(self.tmpdir).find("/new_testcase")
+        self.assertIn("extra-nitrate", fmf_node.data)
+
+    def test_existing(self):
+        fmf_node = Tree(self.tmpdir).find("/existing_testcase")
+        self.assertEqual(fmf_node.data["extra-nitrate"], "TC#0609686")
+        test = tmt.base.Test(fmf_node)
+        print(test.fmf_id)
+        test.export(format_='nitrate', create=True, general=True)
+
+
+class NitrateImport(RequreTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.tmpdir = Path(tempfile.mktemp())
+        shutil.copytree(NITRATE_EXAMPLE, self.tmpdir)
+
+    def tearDown(self):
+        shutil.rmtree(self.tmpdir)
+        super().tearDown()
+
+    def test_import_manual_confirmed(self):
+        runner = CliRunner()
+        # TODO: import does not respect --root param anyhow (could)
+        result = runner.invoke(
+            tmt.cli.main, ['--root', self.tmpdir / "import_case", "test", "import", "--nitrate", "--manual", "--case=609704"])
```

Comment:
Why are you using `self.tmpdir / "import_case"`? Isn't the metadata root just `self.tmpdir`?

Reply:
This is exactly place where I did not understand tmt test import/export, for export it seems to use this `--root` variable for import especially `--manual` it is totally ignored, so it does not any effect what I'll write there.

it is also mentioned in `TODO` comment on previous line

Reply:
I see, for `tmt test import` this should be clarified. Seems we are always expected current working directory.